source/SupportLib/PlayGamesPluginSupport/src/main/java/com/google/games/bridge/TokenFragment.java

```diff
@@ -419,12 +437,20 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     private void onSignedIn(int resultCode, GoogleSignInAccount acct) {
+
+        if (resultCode == CommonStatusCodes.CANCELED) {
+            pendingTokenRequest.cancel();
```

Comment:
I had to add a null check here on my version of the jar. This caused an NPE when the user started the app without GPGS, followed the install prompt, then returned to the app
`09-26 22:07:13.647  6864  6864 E AndroidRuntime: Caused by: java.lang.RuntimeException: Failure delivering result ResultInfo{who=android:fragment:0, request=9002, result=0, data=Intent { (has extras) }} to activity {REDACTED}: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.google.games.bridge.TokenFragment$TokenRequest.cancel()' on a null object reference
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at android.app.ActivityThread.deliverResults(ActivityThread.java:4252)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at android.app.ActivityThread.handleSendResult(ActivityThread.java:4295)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at android.app.ActivityThread.-wrap20(ActivityThread.java)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1583)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:102)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:154)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6290)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
09-26 22:07:13.647  6864  6864 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void com.google.games.bridge.TokenFragment$TokenRequest.cancel()' on a null object reference
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at com.google.games.bridge.TokenFragment.onSignedIn(TokenFragment.java:442)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at com.google.games.bridge.TokenFragment.onActivityResult(TokenFragment.java:424)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at android.app.Activity.dispatchActivityResult(Activity.java:7046)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        at android.app.ActivityThread.deliverResults(ActivityThread.java:4248)
09-26 22:07:13.647  6864  6864 E AndroidRuntime:        ... 9 more
09-26 22:07:13.647  6864  6864 D AppTracker: App Event: crash`

Reply:
Ah great thank you!